### DISCUSSION TITLE
Question about where to find setting - highlight colors

### DISCUSSION THREAD

**Discussion by forelockmocs** [trevorsandy/lpub3d#691] [Q&A] [answered]
When adding (for instance - see picture) tan-coloured pieces, the tan-color of the highlighted piece is much brighter then when it's not highlighted. Is there a setting somewhere to influence the difference between the brightness-difference? I cannot find it...

![image](https://user-images.githubusercontent.com/62996120/227463757-6d71d173-9fbe-4c64-8106-18aecb830551.png)

**Comment by trevorsandy** [comment]
This is an interesting development. It looks like somehow the LDraw COLOUR entry VALUE is also being modified.

In fact, the primary color VALUE should not be affected when parts are highlighted. 

The highlight feature simply modifies the LDraw color's EDGE by changing its value to the highlighted color.

The modified color entry is further updated with a 'Name' (LPub3D_) and color CODE (110) prefix to distinguish it from the original LDraw color.

As an example, for the original Tan color:

`0 !COLOUR Tan CODE  19   VALUE #B0A06F   EDGE #333333`

... a modified !COLOUR entry would be added to the working files, e.g. csi.ldr, which looks like this:

`0 !COLOUR LPub3D_Tan  CODE  11019   VALUE #B0A06F   EDGE #FF33CC`

You can verify these updates by taking a look at the csi.ldr file (for the current step) in the LPub3D working folder. 

In fact, it would be helpful if you could archive (zip) and post a copy of the working folder.

Cheers,

**Comment by trevorsandy** [comment, marked as answer]
FYI, I was not able to reproduce this behaviour with the latest release build.

For me, highlighted parts colour are as expected:

![HighlightColour](https://user-images.githubusercontent.com/13388970/228402625-96e4d0f3-6ef3-479a-8ae9-88b65ecda901.png)

![HighlightColourCsi](https://user-images.githubusercontent.com/13388970/228402927-4a981105-768e-4db3-953a-88068cfe7a3e.png)

Cheers,
